README.rst

```diff
@@ -634,6 +634,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **multiple_unset** [contrib]
```

Comment:
A bit funny - whole codebase follows that, but we don't want to follow that ;)
After said that, let it be contrib indeed.